api/v1alpha1/virtualmachineimage_types.go

```diff
@@ -41,6 +41,26 @@ type VirtualMachineImageOSInfo struct {
 	Type string `json:"type,omitempty"`
 }
 
+// VirtualMachineImageInfo describes basic information related to the image, such as id and name. This information
+// can be used to identify an image in the content source.
+type VirtualMachineImageInfo struct {
+	//ImageID describes the ID of this VirtualMachineImage.
+	ImageID string `json:"imageId"`
+
+	// ImageName describes the display name of this VirtualMachineImage.
+	ImageName string `json:"imageName,omitempty"`
```

Comment:
What situation would this be empty? Generally, such fields also need to be `// +optional` but I'm not sure why this would be `omitempty`